lib/Group_LDAP.php

```diff
@@ -910,4 +910,14 @@ public function implementsActions($actions) {
 	public function isVisibleForScope($scope) {
 		return true;
 	}
+
+	/**
+	 * Returns whether the groups and memberships are to be maintained by core
+	 * and sync mechanism to be used.
+	 *
+	 * @since 10.0.0
```

Comment:
? should this be a reference to cores version, and hence 10.1? or ldap version?

Reply:
the same  as for isVisibleForScope

Reply:
the since annotation is used in core to indicate when an API has been introduced. see `\OCP\GroupInterface`.  So this occurence, https://github.com/owncloud/core/pull/29107/files#diff-2764c6ee76270bcf82f6a5fd186b0ae1R133 and probably other `@since` annotations in that PR should be updated to 10.1. If we backport to 10.0.?